packages/integration-tests/src/components/dom/test-inner-text/integration/inner-text/inner-text.js

```diff
@@ -0,0 +1,23 @@
+import { LightningElement, api } from 'lwc';
+
+export default class InnerText extends LightningElement {
+    _innerText = null;
+
+    @api
+    getTypedText() {
+        return this.template.querySelector('input').value;
+    }
+
+    @api
+    getInnerText() {
+        return this._innerText;
+    }
+
+    computeInputInnerText(evt) {
+        this._innerText = evt.target.innerText;
+    }
+
+    computeDivInnerText(evt) {
+        this._innerText = evt.target.innerText;
```

Comment:
In both cases, `event.target` is an `<input>` element and its not calculating the div's inner text

Reply:
ah shoot, right, i changed this by mistake. fixing it.